### ISSUE TITLE
[a11y] Relay Dashboard low contrast elements

### ISSUE THREAD

**Issue by rbillings** [mozilla/fx-private-relay-add-on#220]
The Relay Dashboard Home page icon fails the WCAG AAA contrast requirements for accessible text. The current contrast is 2.68, [contrast recommendations are here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast). I added all of the elements found by the Firefox accessibility add on, and realize that some of the low contrast was intentional.

Home icon is 2.68:
<img width="176" alt="Screen Shot 2021-11-30 at 1 59 14 PM" src="https://user-images.githubusercontent.com/737878/144134944-f9465def-36d8-4cf3-b10e-4d3e84b05ed6.png">

The signed in banner is 1.87:
<img width="474" alt="Screen Shot 2021-11-30 at 2 04 01 PM" src="https://user-images.githubusercontent.com/737878/144135430-88d73287-77fb-4766-982c-ef3397487167.png">

Get Started steps 1, 2, 3:
<img width="801" alt="Screen Shot 2021-11-30 at 2 04 50 PM" src="https://user-images.githubusercontent.com/737878/144135511-1e25f379-6e4b-4997-964a-3d510d760a73.png">

Action step is 4.15:
<img width="141" alt="Screen Shot 2021-11-30 at 2 11 23 PM" src="https://user-images.githubusercontent.com/737878/144136325-62f8485f-ee6a-4f60-9bc1-e8254256b9a6.png">

'***@' and '.mozmail.com' are 2.68
<img width="454" alt="Screen Shot 2021-11-30 at 2 11 54 PM" src="https://user-images.githubusercontent.com/737878/144136418-6c9afc4c-b57a-43c7-9400-db3abf047cc5.png">

Number and 'Forwarded' are 2.66:
<img width="257" alt="Screen Shot 2021-11-30 at 2 14 23 PM" src="https://user-images.githubusercontent.com/737878/144136691-ce09fa1f-6324-4104-90f6-6f730ba761b9.png"

Delete is 4.46:
<img width="129" alt="Screen Shot 2021-11-30 at 2 15 03 PM" src="https://user-images.githubusercontent.com/737878/144136802-0d98e946-f05a-4017-9058-dca42cdbc477.png">
>
